### ISSUE TITLE
CLI name collision with pslab-python

### ISSUE THREAD

**Issue by bessman** [fossasia/pslab-desktop#694]
* **I'm submitting a ...**
  - [ x] bug report
  - [ ] feature request

* **What is the current status?**
pslab-desktop and pslab-desktop both ship executables named 'pslab'. One or both should be renamed, to for example pslab-gui or pslab-cli.

**Comment by orangecms** [comment]
Yes, we should name this app (configuration is in `package.json`, IIUC) to `pslab-desktop`, just like the repo and what's common among other apps, such as `signal-desktop` etc..

I am creating a symlink to the pslab-desktop binary in https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=pslab-desktop for the same reason.
